src/tests/token/test_erc1155.cairo

```diff
@@ -879,15 +915,30 @@ fn test_update_wac_from_zero_to_non_zero() {
     let recipient = setup_receiver();
     let sender = ZERO();
     let (token_ids, values) = get_ids_and_values();
-    let data = array![].span();
     testing::set_caller_address(owner);
 
     assert_state_before_transfer_from_zero_batch(sender, recipient, token_ids);
 
-    state.update_with_acceptance_check(sender, recipient, token_ids, values, data);
-    assert_only_event_transfer_batch(owner, sender, recipient, token_ids, values);
+    state.update_with_acceptance_check(sender, recipient, token_ids, values, EMPTY_DATA());
+    assert_only_event_transfer_batch(ZERO(), owner, sender, recipient, token_ids, values);
+
+    assert_state_after_transfer_from_zero_batch(sender, recipient, token_ids, values);
+}
+
+#[test]
+fn test_update_wac_from_zero_to_non_zero_camel_receiver() {
```

Comment:
wac sounds like guac to me
![image](https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/3797f946-554b-4532-934a-1b4422b1ad6c)

Reply:
:)